content/post/leadership-ci.md

```diff
@@ -0,0 +1,118 @@
++++
+date = "2019-04-06T8:09:26-07:00"
+title = "Leadership CI"
+author = "Jessica Frazelle"
+description = "Questions to ask yourself to make sure you are in the right mindset to think and behave as a leader."
++++
+
+This post is co-authored by [Kathy Simpson](https://github.com/simpsoka).
+
+> “understanding the true nature of instinctive decision making requires us to be forgiving of those people trapped in circumstances where good judgment is imperiled.” 
+> ― Malcolm Gladwell, Blink: The Power of Thinking Without Thinking
+
+
+As leaders, being consistent in our actions even when under pressure is of
+utmost importance. When writing and delivering software we rely on our
+continuous integration (CI) infrastructure and test suites to tell us when a test is
+failing and code should not be merged.
+
+As leaders, before acting or making decisions it would be nice to have a set of
+tests and checks to make sure we are in the right headspace to think and behave
+in the best interest of everyone.
+
+The following are the sets of questions we brainstormed, but expect them to
+change over time as we experience and deal with new problems. These were
+started in [a gist](https://gist.github.com/simpsoka/14da775a63e22e5083141da5c48e6410)
+and are copied below. The diff of this post and the gist will serve as the
+evolution of this thought process.
+
+It's important to note that in some instances answering all the questions might
+take too much time. Perhaps prioritizing the most important ones in the moment
```

Comment:
Would be cool to make some suggestions about how to put this list into action. Maybe something like.... 

Answering all the questions may be a luxury at times, so we suggest breaking them down based on the situation you find yourself in: prioritize the most important ones to your role, have a few ‘go to’ questions, or categorize them based on the situations you find yourself in more often. The important part of this list is to help us navigate a difficult situation while still maintaining the integrity we intend for ourselves as leaders.

Reply:
oooo I love this :)